includes/preamble.tex

```diff
@@ -1,135 +1,34 @@
 % preamble.tex -- packages to include
-%
-% Wisconsin dissertation template
-% Copyright (c) 2008 William C. Benton.  All rights reserved.
-%
-% This program can redistributed and/or modified under the terms
-% of the LaTeX Project Public License Distributed from CTAN
-% archives in directory macros/latex/base/lppl.txt; either
-% version 1 of the License, or (at your option) any later version.
-%
-% This program includes other software that is licensed under the
-% terms of the LPPL and the Perl Artistic License; see README for details.
-%
-% You, the user, still hold the copyright to any document you produce
-% with this software (like your dissertation).
 
-%% You should use natbib
-%\IfFileExists{natbib.sty}{%
-%\usepackage[numbers,sort&compress]{natbib}
-%%\usepackage{natbib}%
-%}{}
-
-%% You probably need appendix, if you want appendices
-\IfFileExists{appendix.sty}{%
-\usepackage{appendix}%
-}{}
-
-%% the spacing in memoir is weird, you'll need to use this
-\DisemulatePackage{setspace}
-\usepackage[onehalfspacing]{setspace}
-
-%\usepackage{geometry}
-
-%% List setup; the ``hanglist`` environment will allow you to have
-%% nicely-typeset enumerated lists (i.e. with the numbers hanging in
-%% the margins).  You need at least version 2.1 of enumitem.sty.  If
-%% you don't have enumitem installed at all, hanglist will just be an
-%% alias for enumerate.
-\IfFileExists{enumitem.sty}{%
-\usepackage[loadonly]{enumitem}[2007/06/30]%
-\newlist{hanglist}{enumerate}{1}% 
-\setlist[hanglist]{label=\arabic*.}%
-\setlist[hanglist,1]{leftmargin=0pt}%
-}{%
-\newenvironment{hanglist}{\begin{enumerate}}{\end{enumerate}}%
-}
-
-%% Comment out any of these that you don't want
-\usepackage{amssymb}
-\usepackage{amsmath}
-\usepackage{amsthm}
-%\usepackage{theorem}
+%% page/text formatting
+\usepackage[margin=1.0in]{geometry}
 \usepackage{hyperref}
```

Comment:
Do you think it would be helpful for newcomers to have a comment for each package about what features it provides?

Reply:
Yes- thought about doing that and forgot. I can add more comments.